yarn.lock

```diff
@@ -2,85 +2,103 @@
 # yarn lockfile v1
 
 
-"@apideck/better-ajv-errors@^0.3.1":
```

Comment:
Please put this file in `.gitignore` and unstage this file. There is no need for this file to be on github

Reply:
I am not able to gitignore the yarn.lock. It is their in the file but not ignoring it.


![Screenshot (50)](https://user-images.githubusercontent.com/76507095/178132114-0987369a-9178-4ceb-9cb2-44df9d706eca.png)